esp/templates/themes/editor.html

```diff
@@ -384,6 +409,13 @@ <h3 class="accordion-toggle" data-toggle="collapse" href="#advanced" rel="popove
                             <h3>Category: {{ category }}<hr/></h3>
                             <!-- Dropdown box for adding an optional variable override that is not yet defined -->
                             <div class="control-group opt_var_div hidden">
+                                <div> Accent 1 for each side tab refers to the background color of the heading bar and the top tabs when the specified number side tab is selected.
```

Comment:
Accent 1 looks to be used for the unselected top tabs? Also, it looks like Accent 1 is used for the little bit on the side tab?